src/Parser/RestoDruid/Modules/Legendaries/Sephuz.js

```diff
@@ -7,12 +10,19 @@ const ONE_PERCENT_HASTE_RATING = 375;
 const SEPHUZ_PROCC_HASTE = 25;
 const SEPHUZ_BUFF_ID = 208052;
 
-class Sephuz extends Module {
+class Sephuz extends CoreSephuz {
+  static dependencies = {
+    combatants: Combatants,
+  };
   uptime = 0;
   throughput = 0;
   sephuzProccInHasteRating = SEPHUZ_PROCC_HASTE * ONE_PERCENT_HASTE_RATING;
   sephuzStaticHasteInRating = 0;
+
   on_initialized() {
+    if (!this.owner.error) {
```

Comment:
This is outdated and should be removed. `this.owner.error` is completely useless now, could you remove this all over the codebase please?

Reply:
Meaning it can't ever happen so anything that's inside these conditionals should be outside it? And do you mean in all the parsers or just my Warlocks? (No problem if all, just to clarify "codebase" :D )

Reply:
All of them, I went through everything once but someone must have been working on stuff that still had it and reintroduced it xD